.gitignore

```diff
@@ -39,3 +39,7 @@ node_modules
 .floobit
 .floo
 .flooignore
+
```

Comment:
Consider also adding in bundle.js for gitignore as webpack will automatically create a new one each time you run it.  Pushing up a new bundle.js each time will also create unnecessary merge conflicts

Reply:
ah thanks for this